tests/test_cli.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from click.testing import CliRunner
+from getgist.__main__ import (
+    run_getgist,
+    run_getmy,
+    run_putgist,
+    run_putmy,
+    run_lsgists,
+    run_mygists,
+)
+
+runner = CliRunner()
+
+
+@pytest.mark.parametrize(
+    "command, to_patch, args",
+    [
+        (run_getmy, None, []),
+        (run_putmy, "getgist.__main__.GetGist.put", []),
```

Comment:
Why do we need to patch it? Shouldn't the error stop the execution before that point?